packages/expo-av/package.json

```diff
@@ -1,41 +1,47 @@
 {
-  "name": "expo-av",
-  "version": "9.2.0",
-  "description": "Expo universal module for Audio and Video playback",
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
-  "sideEffects": false,
-  "scripts": {
-    "build": "expo-module build",
-    "clean": "expo-module clean",
-    "lint": "expo-module lint",
-    "test": "expo-module test",
-    "prepare": "expo-module prepare",
-    "prepublishOnly": "expo-module prepublishOnly",
-    "expo-module": "expo-module"
-  },
-  "keywords": [
-    "expo",
-    "react-native",
-    "audio",
-    "video"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/expo/expo.git",
-    "directory": "packages/expo-av"
-  },
-  "bugs": {
-    "url": "https://github.com/expo/expo/issues"
-  },
-  "author": "650 Industries, Inc.",
-  "license": "MIT",
-  "homepage": "https://docs.expo.io/versions/latest/sdk/av/",
-  "dependencies": {
-    "@expo/config-plugins": "^3.0.0",
-    "expo-modules-core": "~0.1.1"
-  },
-  "devDependencies": {
-    "expo-module-scripts": "^2.0.0"
-  }
-}
+	"name": "expo-av",
+	"version": "9.2.0",
+	"description": "Expo universal module for Audio and Video playback",
+	"main": "build/index.js",
+	"types": "build/index.d.ts",
+	"sideEffects": false,
+	"scripts": {
+		"build": "expo-module build",
+		"clean": "expo-module clean",
+		"lint": "expo-module lint",
+		"test": "expo-module test",
+		"prepare": "expo-module prepare",
+		"prepublishOnly": "expo-module prepublishOnly",
+		"expo-module": "expo-module",
+		"start:examples": "et run-stories"
+	},
+	"keywords": [
+		"expo",
+		"react-native",
+		"audio",
+		"video"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/expo/expo.git",
+		"directory": "packages/expo-av"
+	},
+	"bugs": {
+		"url": "https://github.com/expo/expo/issues"
+	},
+	"author": "650 Industries, Inc.",
+	"license": "MIT",
+	"homepage": "https://docs.expo.io/versions/latest/sdk/av/",
+	"dependencies": {
+		"@expo/config-plugins": "^3.0.0",
+		"expo-modules-core": "~0.1.1"
+	},
+	"devDependencies": {
+		"expo-module-scripts": "^2.0.0"
+	},
+	"expoStories": {
+		"packages": {
+			"expo-av": "~9.2.0"
+		}
+	}
+}
```

Comment:
Please revert indent changes in this file. We aim to indent everything with 2 spaces in every file kind.